e2e/common/idamLoginHelper.ts

```diff
@@ -60,6 +60,24 @@ export class IdamLoginHelper {
     await page.fill(this.fields.password, userInfo.password);
     await page.click(this.submitButton);
   }
+
+  public static async signInJudgeUser(
+    page: Page,
+    application: string,
+  ): Promise<void> {
+    if (!page.url().includes("idam-web-public.")) {
+      await page.goto(application);
+    }
+
+    const userCredentials: UserCredentials = Config.getUserCredentials("judge");
+    if (userCredentials) {
+      await page.fill(this.fields.username, userCredentials.email);
```

Comment:
these fill actions are now repeated a few times, maybe future work need to refactor to reduce duplicated code